cmd/exo/cmd/vm_create.go

```diff
@@ -302,7 +306,7 @@ func saveKeyPair(keyPairs *egoscale.SSHKeyPair, vmID string) {
 func init() {
 	vmCreateCmd.Flags().StringP("cloud-init-file", "f", "", "Deploy instance with a cloud-init file")
 	vmCreateCmd.Flags().StringP("zone", "z", "", "<zone name | id | keyword> (ch-dk-2|ch-gva-2|at-vie-1|de-fra-1)")
-	vmCreateCmd.Flags().StringP("template", "t", "Linux Ubuntu 18.04", "<template name | id>")
+	vmCreateCmd.Flags().StringP("template", "t", "", fmt.Sprintf("<template name | id> (default: %s)", defaultTemplate))
```

Comment:
It's wrong it's always be default const value and config file value ! cobra is executed before config 😉

Reply:
I guess I fixed it in the meantime.

Reply:
try and you will see, when you do `exo vm create -h` Cobra execute cmd tree before execute root.go config code

Reply:
I tried your code an hour ago

Reply:
big `T` :)